### ISSUE TITLE
Logo of page should be h1 and have text

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#361]
In order for somebody to read the logo without a visual display, it should be an H1 and have some sort of accessible text.

>
Heading tag found with no content. Text that is not intended as a heading should not be marked up with heading tags.

>
The heading structure is not logically nested. This h2 element appears to be the primary document heading, so should be an h1 element.

**Comment by msecret** [comment]
Solution idea:

```
<h1>
  <svg>
    <title>cloud.gov</title>
  </svg>
  <span class="accessible_text">Cloud.gov</span>
</h1>

.accessible_text {
  position: absolute;
  left: -999999999px;
}
```

**Comment by msecret** [comment]
Alternate solution:

```
<svg class="logo">
</svg>
<h1 class="accessible_text">cloud.gov</h1>
```

**Comment by jeremiak** [comment]
Looks fixed with d310d2bd46449af223e00e17413db7307b24b8de
